frontend/src/core/resource/components/ResourceMenu.js

```diff
@@ -93,6 +115,29 @@ export class ResourceMenuBase extends React.Component<Props, State> {
             resource.path.toLowerCase().indexOf(search.toLowerCase()) > -1
         );
 
+        const sort = this.state.sortAsc ? "fa fa-caret-up" : "fa fa-caret-down";
+
+        const sortResource = this.state.sortActive === 'resource' ? sort : ""
+
+        const sortProgress = this.state.sortActive === 'progress' ? sort : "";
```

Comment:
Please:
* always use single quotes
* always use colons at the end of lines
* remove both blank lines 

Rename:
* sortResource -> resourceClass
* sortProgress -> progressClass